src/displayapp/screens/WatchFaceAnalog.cpp

```diff
@@ -49,22 +49,28 @@ WatchFaceAnalog::WatchFaceAnalog(Pinetime::Applications::DisplayApp* app,
                                  Controllers::Battery& batteryController,
                                  Controllers::Ble& bleController,
                                  Controllers::NotificationManager& notificationManager,
-                                 Controllers::Settings& settingsController)
+                                 Controllers::Settings& settingsController,
+                                 Controllers::HeartRateController& heartRateController,
+                                 Controllers::MotionController& motionController)
   : Screen(app),
     currentDateTime {{}},
     dateTimeController {dateTimeController},
     batteryController {batteryController},
     bleController {bleController},
     notificationManager {notificationManager},
-    settingsController {settingsController} {
+    settingsController {settingsController},
+    heartRateController {heartRateController},
+    motionController {motionController} {
 
   sHour = 99;
   sMinute = 99;
   sSecond = 99;
 
-  lv_obj_t* bg_clock_img = lv_img_create(lv_scr_act(), NULL);
-  lv_img_set_src(bg_clock_img, &bg_clock);
-  lv_obj_align(bg_clock_img, NULL, LV_ALIGN_CENTER, 0, 0);
+  stepsMeter = lv_linemeter_create(lv_scr_act(), NULL);
+  lv_linemeter_set_range(stepsMeter, 0, 12);
+  lv_linemeter_set_scale(stepsMeter, 360, 13);
+  lv_obj_set_size(stepsMeter, 250, 250);
+  lv_obj_align(stepsMeter, NULL, LV_ALIGN_CENTER, 0, 0);
```

Comment:
This object goes beyond the screen area, yet it doesn't fill the entire screen. Maybe there's an invisible background. The clock looks a bit small for the screen, so this should be made as large as possible.
![image](https://user-images.githubusercontent.com/37774658/169515214-455c8d45-394f-4c58-8e40-d12731e8754b.png)

Reply:
Started looking into this, I have set the background to red in order to see it, and indeed there is a background that is larger than the indicators / rendered lines. However, I have tried setting the style with several properties but I cannot get the background to shrink...

```
  lv_obj_set_style_local_bg_color(stepsMeter, LV_LINEMETER_PART_MAIN, LV_STATE_DEFAULT, LV_COLOR_RED);
  lv_obj_set_style_local_pad_inner(stepsMeter, LV_LINEMETER_PART_MAIN, LV_STATE_DEFAULT, 0);
  lv_obj_set_style_local_margin_left(stepsMeter, LV_LINEMETER_PART_MAIN, LV_STATE_DEFAULT, 0);
```
Will keep poking away at it, but the larger than resolution was the quick and dirty as you have pointed out.